packages/sit-onyx/src/components/OnyxBasicDialog/types.ts

```diff
@@ -31,6 +31,11 @@ export type OnyxBasicDialogProps = DensityProp & {
    * How to align the dialog horizontally. If "left" or "right", the modal will also be made full height.
    */
   alignment?: DialogAlignment;
+  /**
+   * The intensity of the backdrop.
+   * @default "light"
+   */
+  backdrop?: "light" | "intense";
```

Comment:
light might be confusing when its used in dark mode 🤔 
What do you think about aligning them with the CSS variable names that are used for the modes so to call them "soft" and "medium"